policy-controller/k8s/status/src/index.rs

```diff
@@ -173,14 +179,19 @@ impl Index {
         true
     }
 
-    fn parent_status(&self, parent_ref: &ParentReference) -> Option<gateway::RouteParentStatus> {
+    fn parent_status(
+        &self,
+        parent_ref: &ParentReference,
+        backend_condition: &k8s::Condition,
```

Comment:
no point in passing the backend_condition here by reference if we're just going to clone it.  May as well just pass it in by value.